### PR TITLE
Use HTTPS when fetching

### PR DESCRIPTION
**Pull request by thughes**
See dodo/node-unicodetable#27 and https://nodesecurity.io/advisories/161

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "node": ">= 0.8.x"
   },
   "scripts": {
-    "download": "curl \"http://unicode.org/Public/9.0.0/ucd/UnicodeData.txt\" > UnicodeData.txt",
+    "download": "curl \"https://unicode.org/Public/9.0.0/ucd/UnicodeData.txt\" > UnicodeData.txt",
```
Comment (tdanecker) [inline package.json:19]: have you tried the url? unfortunatly unicode.org doesn't support https :disappointed:

**Review by tdanecker — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (thughes) [reply package.json:19]: No, unfortunately the URL is blocked for me. I naively assumed a site like this would support HTTPS these days...

Comment (tdanecker) [reply package.json:19]: me too. I already wanted to approve the pr ;) thx for the contribution anyways, though!

**Comment by alexframe** [comment]
@tdanecker this is now supported over HTTPS, could this be merged?

https://unicode.org/Public/9.0.0/ucd/UnicodeData.txt

**Comment by alexframe** [comment]
@tdanecker can we merge this now please?

It will resolve [node security platform advisory 161](https://nodesecurity.io/advisories/161)

**Comment by tdanecker** [comment]
Hi,

I'm just back from holidays. The changes are merged now! Though the UnicodeData.txt isn't downloaded at installation-time anymore but only before the package is published. It didn't effect any users of the package, just the maintainers ;)

But thx for the hint anyways! It was about time that unicode.org supported https!

Cheers

**Comment by mrgordon** [comment]
node projects still can't pass security scans because the latest `npm` includes `unicode` 6.3.0 which still has this http vulnerability. Any chance we can get this upgraded soon upstream? Not sure how much insight you have into which version of `unicode` they are building on

The exact dependency on this is via

`/usr/lib/node_modules/npm/node_modules/libnpx/node_modules/yargs/node_modules/read-pkg-up/node_modules/read-pkg/node_modules/load-json-file/node_modules/parse-json/vendor`

**Comment by mrgordon** [comment]
Alternatively I was considering manually upgrading the version of `unicode` that is used. Is a changelog available anywhere? I'm trying to gauge how risky it is to go from 6.3.0 to 9.0.0 where the vulnerability is resolved.

**Comment by tdanecker** [comment]
The new releases should be fully compatible with old versions.

Previously, the most recent data has been downloaded from unicode.org during install-time. Now the version-schema matches the unicode.org releases (which are published about once a year) and includes exactly that version of the data. But that's all that changed.

I would recommend to upgrade to the most recent version (10.0.0) for up-to-date unicode data.

For changes in the data, just have a look at the unicode.org release notes.

**Pull request closed without merging**